frontend/ui/widget/iconbutton.lua

```diff
@@ -92,20 +92,24 @@ function IconButton:initGesListener()
 end
 
 function IconButton:onTapIconButton()
-    UIManager:scheduleIn(0.0, function()
-        self.image.invert = true
-        UIManager:setDirty(self.show_parent, function()
-            return "ui", self[1].dimen
-        end)
-    end)
-    -- make sure button reacts before doing callback
-    UIManager:scheduleIn(0.1, function()
+    if G_reader_settings:isFalse("flash_ui") then
         self.callback()
-        self.image.invert = false
-        UIManager:setDirty(self.show_parent, function()
-            return "ui", self[1].dimen
+    else
+        UIManager:scheduleIn(0.0, function()
```

Comment:
Is there a functional difference with nextTick?

Reply:
Apologies for the confusing order of comments. But yeah, I think this scheduleIn(0.0 is probably just a legacy thing?

Reply:
It's like that in all these _scheduled invert and scheduled put it back_. It was probably there in the first one, then people cut & pasted from it, like @robert00s did yesterday when he added it to KeyValuePage. I wouldn't bother updating all the existing code, as it causes no problem, and it's just what nextTick does I think :)

Reply:
But if you're updating it anyway you might as well update the legacy stuff. :-P